docs/concepts/lifecycle.puml

```diff
@@ -1,7 +1,86 @@
 @startuml
-start
-:Draft;
-:Active;
-:Closed;
-stop
+partition #B0E3E6 "Funnel" {
+
+    note right
+      * there is an <b>owner</b>
+      * there is a <b>lead</b> e.g. proposed via owner, BalMATE, ...
+      * the topic has <b>severity</b> via votes
+      ====
+      <b>OnePager</b> from owner 
+      about high-level <b>scope and goal</b>
+      as <b>Epic in JIRA</b>
+    end note
+
+    partition #LightGrey "Handle new Business Request" {
+        #LightGrey:Create Request;
+        #LightGrey:Community Voting;
```

Comment:
The notes on the right seem to apply for "Create Request" only. I am missing examples / calrification for Voting and Traige & Assign

Reply:
I agree - from our discussions so far we agreed (imho) on taking this as a given input, which is also why 
 - the process starts within refine
 - all boxes in handle new business request are #LightGrey
 - the examples cover what we've discussed to far